styles.css

```diff
@@ -181,4 +704,170 @@ ul.navbar-items {
     background-size: 250px;
     background-repeat: no-repeat;
   }
+
+  /* works-section responsive style  */
+
+  #works-section {
+    margin-bottom: 100px;
+  }
+
+  .box-container {
+    flex-direction: column;
+    margin: 5px 10px;
+  }
+
+  .left-section {
+    margin-bottom: 20px;
+  }
+
+  .left-section img {
+    width: 100%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 10px;
+    margin: 0;
+    padding-left: 4%;
+    padding-right: 4%;
+  }
+
+  .first-desc {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .first-desc h3 {
+    font-size: 16px;
+  }
+
+  .first-desc ul {
+    padding-left: 0;
+  }
```

Comment:
- [ ] Please 🙏🏽 correct the margin between the elements and correct the button size to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Xghayor/Portfolio/assets/58642949/1290ff6d-7c88-4227-b3fc-4a9efcfe9627) | ![image](https://github.com/Xghayor/Portfolio/assets/58642949/a6019df0-a074-44c5-ae6b-927d8100493e) |